libraries/AP_HAL_Linux/RCOutput_Raspilot.cpp

```diff
@@ -50,11 +43,12 @@ void RCOutput_Raspilot::set_freq(uint32_t chmask, uint16_t freq_hz)
     _dma_packet_tx.regs[0] = freq_hz;
     _dma_packet_tx.crc = 0;
     _dma_packet_tx.crc = crc_packet(&_dma_packet_tx);
-    _spi->transaction((uint8_t *)&_dma_packet_tx, (uint8_t *)&_dma_packet_rx, sizeof(_dma_packet_tx));
+    _dev->transfer((uint8_t *)&_dma_packet_tx, sizeof(_dma_packet_tx),
+		   (uint8_t *)&_dma_packet_rx, sizeof(_dma_packet_rx));
```

Comment:
/me wondering why this was called "dma_packet" in the first place. It has nothing to do with dma.